tests/fuzz/Makefile

```diff
@@ -22,10 +22,18 @@ ifndef LIBFUZZER_ROOT
     export LIBFUZZER_ROOT = $(shell echo "../../fuzz_dependencies")
 endif
 
+ifndef FUZZ_TIMEOUT_SEC
+    export FUZZ_TIMEOUT_SEC=120
+endif
+
+ifndef FUZZ_TESTS
+    export FUZZ_TESTS=${TESTS}
+endif
+
 
 .PHONY : all
 all : ld-preload
-all : $(TESTS)
+all : run_tests
```

Comment:
Is ld-preload is a prereq for run_tests? Might be better to do `all: run_tests` and `run_tests $(TESTS) ld-reload` As is, if we ever tried to run this in parallel run-tests might start before ld-preload is done.